cmd/entrypoint/waiter.go

```diff
@@ -42,7 +43,12 @@ func (rw *realWaiter) Wait(file string, expectContent bool) error {
 		} else if !os.IsNotExist(err) {
 			return fmt.Errorf("waiting for %q: %w", file, err)
 		}
+		// Do not return skipError when breakpointOnFailure is enabled as it breaks
```

Comment:
How does it break it? Can you add more comment here?

Reply:
Added the following comment 
```
// When a .err file is read by this step, it means that a previous step has failed
// We wouldn't want this step to stop executing because the previous step failed during debug
// That is counterproductive to debugging
// Hence we disable skipError here so that the other steps in the failed taskRun can continue
// executing if breakpointOnFailure is enabled for the taskRun
// TLDR: Do not return skipError when breakpointOnFailure is enabled as it breaks execution of the TaskRun
```
LMK WYT